config/environments/development.rb

```diff
@@ -26,11 +26,22 @@
     config.cache_store = :null_store
   end
 
+  config.active_job.queue_adapter = :sidekiq
+
+  config.action_mailer.perform_deliveries = true
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
 
   config.action_mailer.perform_caching = false
-
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => ENV["SMTP_USERNAME"],
+    :password => ENV["SMTP_PASSWORD"],
+    :address => ENV["SMTP_ADDRESS"],
+    :domain => ENV["SMTP_DOMAIN"],
+    :port => ENV["SMTP_PORT"],
+    :authentication => :cram_md5
```

Comment:
the `ENV['xxxx']` must be change by a secrets.yml value like `Rails.application.secrets.smtp_username`

Reply:
Done

Reply:
Done